PixelHunter1995/Content/Scenes/full_club_room.tmx

```diff
@@ -1,15 +1,58 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<map version="1.4" tiledversion="1.4.1" orientation="orthogonal" renderorder="right-down" width="712" height="160" tilewidth="1" tileheight="1" infinite="0" nextlayerid="9" nextobjectid="8">
+<map version="1.4" tiledversion="1.4.1" orientation="orthogonal" renderorder="right-down" width="712" height="160" tilewidth="1" tileheight="1" infinite="0" nextlayerid="9" nextobjectid="15">
+ <properties>
+  <property name="init" value="IntroInit"/>
+ </properties>
  <tileset firstgid="1" source="../Tileset/dogs.tsx"/>
  <tileset firstgid="65" source="../Tileset/npc_dogs.tsx"/>
  <imagelayer id="2" name="background">
   <image source="../Images/full_club_room.png" width="712" height="160"/>
  </imagelayer>
- <objectgroup id="4" name="dogs">
-  <object id="2" gid="1" x="479.091" y="98.1818" width="32" height="32"/>
+ <objectgroup color="#ff0000" id="4" name="dogs">
+  <object id="2" gid="1" x="501.091" y="103.182" width="32" height="32"/>
   <object id="3" gid="2" x="379.636" y="89.6364" width="32" height="32"/>
   <object id="4" gid="3" x="46.5455" y="125.455" width="32" height="32"/>
-  <object id="7" gid="65" x="120" y="139" width="32" height="48"/>
+  <object id="7" gid="65" x="120" y="139" width="32" height="48">
+   <properties>
+    <property name="item(charcoalPencils)" value="IntroNickCharcoal"/>
+    <property name="item(chips)" value="IntroNickChips"/>
+    <property name="item(conk)" value="IntroNickConk"/>
+    <property name="item(fork)" value="IntroNickFork"/>
+    <property name="use" value="IntroNick"/>
+   </properties>
+  </object>
+  <object id="8" gid="69" x="184" y="106" width="32" height="48">
+   <properties>
+    <property name="use" value="IntroUrsula"/>
+   </properties>
+  </object>
+  <object id="9" gid="68" x="414" y="126" width="32" height="48">
+   <properties>
+    <property name="use" value="IntroAlex"/>
+   </properties>
+  </object>
+  <object id="10" x="10.75" y="65.75">
+   <properties>
+    <property name="look" value="IntroWindow"/>
```

Comment:
I think the mapping should be tied to the action that is being performed, to make easily overviewed distinction between actions. i.e. IntroLookWindow for look, IntroPickupWindow for pickup etc.

Reply:
I updated to names to be more explicit.